content/tracing/advanced_usage/_index.md

```diff
@@ -1324,6 +1415,13 @@ Coming Soon. Reach out to [the Datadog support team][contact support] to be part
 
 [contact support]: https://docs.datadoghq.com/help
 
+{{% /tab %}}
+{{% tab "C++" %}}
+
+Coming Soon. Reach out to [the Datadog support team][contact support] if you're interested in this feature.
```

Comment:
Do we need this? Can we just not have the tab?

Reply:
Cargo-culted this, since it seems very common on this page. Removing the tab entirely causes no errors, it shows the 'Java' tab by default. Perhaps that's a bit confusing, since changing one tab changes all the tabs on a page. So once 'C++' is clicked once, the user will expect all blocks of text to be pertinent to C++. I think an empty tab would be better? 

I'll put up a change with an empty tab, but if you still prefer no tab/show Java instead that's also fine :)

Reply:
I changed my mind! I saw some more cases where the tab is just missing entirely (PHP often), so since that's already a pattern I'll use it too.